src/components/ControlLink/ControlLink.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Clickable, ClickableProps } from '../Clickable';
+
+import './ControlLink.scss';
+
+export type ControlLinkProps = ClickableProps & {
+  color?: 'cement' | 'ink';
+  disabled?: boolean;
+  underlined?: boolean;
+};
+
+export const ControlLink: React.FC<ControlLinkProps> = ({
+  className,
+  color = 'cement',
+  disabled,
+  children,
+  underlined,
+  onClick
+}) => (
+  <Clickable
+    className={classNames(
+      'ControlLink',
+      { 'ControlLink--underlined': underlined },
+      { 'ControlLink--disabled': disabled },
+      `ControlLink--${color}`,
```

Comment:
I'd prefer not to embed the color in the class name.  The third segment in BEM should be a logical state, rather than a color.  So, I'd think that the values should be `resting|hover|focus|disabled`.

Reply:
"Underlined" can perhaps be the Element?  Like, `ControlLink__underlined--resting`, perhaps?  It's weird, but that fits how it's laid out in the figma file.

Reply:
`underlined` is definitely a modifier, I think `--resting` is not required, you can just do `:hover, :focus` or `--hover`, `--focus` if you need that in js altho I think you probably don't.
See how the `Button` is done btw.